src/scripts/achievements/AchievementHandler.ts

```diff
@@ -242,6 +242,11 @@ class AchievementHandler {
         AchievementHandler.addAchievement('Don\'t stop trying', 'Complete 2500 total stages in Battle Frontier', new BattleFrontierTotalStageRequirement(2500), 0.25);
         AchievementHandler.addAchievement('King of the hill', 'Complete 5000 total stages in Battle Frontier', new BattleFrontierTotalStageRequirement(5000), 0.4);
 
+        AchievementHandler.addAchievement('Let\'s try this out', 'Obtain your first Protein', new ProteinObtainRequirement(1), 0.20);
+        AchievementHandler.addAchievement('Training progress', 'Obtain 10 Proteins', new ProteinObtainRequirement(10), 0.30);
+        AchievementHandler.addAchievement('Strong like the mountain', 'Obtain 100 Proteins', new ProteinObtainRequirement(100), 0.40);
+        AchievementHandler.addAchievement('A literal mountain of muscle', 'Obtain 1000 Proteins', new ProteinObtainRequirement(1000), 0.50);
```

Comment:
I feel like the achievement bonus amounts should be reduced a bit.

We should probably add some higher amounts too, maybe similar to BF total stages achievements, but up to 10k?

Reply:
Maybe bonuses at every 1 and 5 (so 1,5,10,50,100,500,1k,5k,10k), and have the bonus start lower (like 0.05 or 0.01) and finish at 0.5?